resources/profiles/Zonestar.ini

```diff
@@ -383,289 +681,514 @@ machine_max_jerk_y = 10
 retract_before_wipe = 0%
 retract_layer_change = 0
 retract_length = 6.5
 retract_length_toolchange = 0
-retract_lift = 0
 retract_lift_above = 0
 retract_lift_below = 0
 retract_restart_extra = 0
 retract_restart_extra_toolchange = 0
-retract_speed = 40
-start_gcode = M104 S[first_layer_temperature] ; set extruder temp\nM140 S[first_layer_bed_temperature] ; set bed temp\nM190 S[first_layer_bed_temperature] ; wait for bed temp\nM109 S[first_layer_temperature] ; wait for extruder temp\nG28 ; home all axes\nG1 Z5 F3000 ; lift nozzle\nG1 X5 Y10 F1500 ; move to prime\nG1 Z0.2 F3000 ; get ready to prime\nG92 E0 ; reset extrusion distance\nG1 Y200 E15 F600 ; prime nozzle\nG1 Y100 F5000 ; quick wipe\nG92 E0
+retract_speed = 30
 use_relative_e_distances = 0
+wipe = 0
+max_volumetric_speed = 5
+single_extruder_multi_material = 0
 
-#Z9 with different types of hotend
-[printer:Zonestar Z9+M3 HOTEND]
-inherits = *common*
-printer_model = Z9
+[printer:*DDE @0.2mm nozzle*]
+inherits = *DDE*
+printer_variant = 0.2
+nozzle_diameter = 0.2
+min_layer_height = 0.05
+max_layer_height = 0.18
+default_print_profile = 0.2 nozzle default
+
+[printer:*DDE @0.3mm nozzle*]
+inherits = *DDE*
+printer_variant = 0.3
+nozzle_diameter = 0.3
+min_layer_height = 0.06
+max_layer_height = 0.28
+default_print_profile = 0.3 nozzle default
+
+[printer:*DDE @0.4mm nozzle*]
+inherits = *DDE*
 printer_variant = 0.4
-bed_shape = 0x0,310x0,310x310,0x310
-default_filament_profile = Generic PLA @Zonestar
-default_print_profile = 0.20mm @Zonestar MULTI
+nozzle_diameter = 0.4
+min_layer_height = 0.1
+max_layer_height = 0.36
+default_print_profile = 0.4 nozzle default
+
+[printer:*DDE @0.6mm nozzle*]
+inherits = *DDE*
+printer_variant = 0.6
+nozzle_diameter = 0.6
+min_layer_height = 0.2
+max_layer_height = 0.5
+default_print_profile = 0.6 nozzle default
+
+[printer:*DDE @0.8mm nozzle*]
+inherits = *DDE*
+printer_variant = 0.8
+nozzle_diameter = 0.8
+min_layer_height = 0.3
+max_layer_height = 0.72
+default_print_profile = 0.8 nozzle default
+
+[printer:*DDE @1.0mm nozzle*]
+inherits = *DDE*
+printer_variant = 1.0
+nozzle_diameter = 1.0
+min_layer_height = 0.4
+max_layer_height = 0.9
+default_print_profile = 1.0 nozzle default
+
+[printer:*DDE @1.2mm nozzle*]
+inherits = *DDE*
+printer_variant = 1.2
+nozzle_diameter = 1.2
+min_layer_height = 0.5
+max_layer_height = 1.1
+default_print_profile = 1.2 nozzle default
+
+## Extruders >> R2 Extruder
+[printer:*R2*]
+num_extruders = 2
+deretract_speed = 30,30
+extruder_colour = #FF0000;#00FF00
+extruder_offset = -9,9
+retract_before_travel = 8,8
+retract_lift = 1.0,1.0
+retract_before_wipe = 0%,0%
+retract_layer_change = 0,0
+retract_length = 8,8
+retract_length_toolchange = 8,8
+retract_lift_above = 0,0
+retract_lift_below = 0,0
+retract_restart_extra = 0,0
+retract_restart_extra_toolchange = 0.2,0.2
+retract_speed = 35,35
+use_relative_e_distances = 0
+wipe = 0,0
+max_volumetric_speed = 5
+single_extruder_multi_material = 0
+ooze_prevention = 1
```

Comment:
This key does not belong to printer section. Remove it from all printer sections.

You can run prusaslicer with --loglevel=5 parameter to see all errors.